### DISCUSSION TITLE
gatsby develop MAC M1 Something went wrong installing the “sharp” module

### DISCUSSION THREAD

**Discussion by Florian-crg** [gatsbyjs/gatsby#31439] [Help]
Running npm run dev (gatsby develop) on MacBook Pro M1 chip exit with   `Error: Something went wrong installing the "sharp" module`

Running npm run dev MacBook Pro intel chip works fine.

<img width="1114" alt="Capture d’écran 2021-05-16 à 20 31 56" src="https://user-images.githubusercontent.com/20863963/118408396-c364c980-b685-11eb-8f6a-18faf4b2e187.png">

https://stackoverflow.com/questions/67560211/gatsby-develop-mac-m1-something-went-wrong-installing-the-sharp-module

**Comment by LekoArts** [comment]
Hey, please read the other already existing discussion on this, you should find your solution there: https://github.com/gatsbyjs/gatsby/discussions?discussions_q=m1+mac
